assets/src/components/parts/janus-popup/Popup.tsx

```diff
@@ -22,6 +22,7 @@ const Popup: React.FC<any> = (props) => {
   const [popupVisible, setPopupVisible] = useState(true);
   const [iconSrc, setIconSrc] = useState('');
 
+  const [initSnapshot, setInitSnapshot] = useState<any>();
```

Comment:
Is there a more specific interface than `any` that could be used here?

Reply:
maybe it could be `Record<string, any>`

Reply:
Or `Record<string, unknown>`

Reply:
Done

Reply:
I was planning using this

 snapshot: { [key: string]: string };

but something is happening with my Git. Hang On..

Reply:
`Record<string, string>` would be the same as that, but can't it have other values than string? `Record<string, unknown>` sounds good